README.md

```diff
@@ -56,10 +71,186 @@ These three files can be downloaded directly from `inputs` and `dataset` directo
 
 Refer to [installation section](#installation) to set-up DeezyMatch on your local machine. 
 
-:warning: In the following tutorials, we assume a directory structure specified in the [this section](#data-and-directory-structure-in-tutorials).
+:warning: In the following tutorials, we assume a directory structure specified in [this section](#data-and-directory-structure-in-tutorials).
  
  Written in the Python programming language, DeezyMatch can be used as a stand-alone command-line tool or can be integrated as a module with other Python codes. In what follows, we describe DeezyMatch's functionalities in different examples and by providing both command lines and python modules syntaxes.
 
+### Quick tour 
+
+In this "quick tour", we go through all the DeezyMatch main functionalities with minimal explanations. If you want to know more about each module, refer to the relevant part of README (also referenced in this section):
+
+:warning: In the following tutorials, we assume a directory structure specified in [this section](#data-and-directory-structure-in-tutorials).
```

Comment:
You have the same warning twice in a few lines, I think the first one is enough, but no problem if you prefer keeping it

Reply:
Removed